python/tox.ini

```diff
@@ -40,7 +40,8 @@ deps =
     -r{toxinidir}/requirements_lint.txt
 commands =
     # flake8 includes black check due to flake8-black
-    flake8 pynessie tests tools
+    # ANN101+102 should not be enabled by default, see https://github.com/sco1/flake8-annotations/issues/75
+    flake8 --extend-ignore=ANN101,ANN102 pynessie tests tools
```

Comment:
Can we just add it to `setup.cfs` https://github.com/projectnessie/nessie/blob/main/python/setup.cfg#L20 ?

Reply:
yes we can, i did not realize flake8 was configured here, since was only looking into `pyproject.toml` and `.pylintrc` :sweat_smile: